Makefile

```diff
@@ -74,6 +77,22 @@ override _HLPFMT = "%-20s %s\n"
 # N/B: There are length/character limitations in GCE for image names
 IMG_SFX ?=
 
+# Env. vars needed by packer
+export CHECKPOINT_DISABLE = 1  # Disable hashicorp phone-home
+export PACKER_CACHE_DIR = $(call err_if_empty,_TEMPDIR)
+
+# Vars from image_info.json for various targets
+override _fedora_release = $(call get_image_info,fedora.release)
+override _fedora_image_url = $(call get_image_info,fedora.image_url)
+override _fedora_csum_url = $(call get_image_info,fedora.csum_url)
+override _fedora_ami = $(call get_image_info,fedora.ami)
+override _fedora_arm64_ami = $(call get_image_info,fedora.arm64_ami)
+override _prior_fedora_release = $(call get_image_info,prior_fedora.release)
+override _prior_fedora_image_url = $(call get_image_info,prior_fedora.image_url)
+override _prior_fedora_csum_url = $(call get_image_info,prior_fedora.csum_url)
+override _ubuntu_release = $(call get_image_info,ubuntu.release)
+override _ubuntu_base_family = $(call get_image_info,ubuntu.family)
+
```

Comment:
Just from a cursory first-pass review, I find this (plus the `get` script) kind of fragile and hard to read/maintain. My vote is to ditch the separate yaml and script.

Reply:
Ya that's what I was thinking too. Okay I'll simplify that into the Makefile then.